spec/takeaway_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'takeaway'
+require 'current_order'
+
+describe Takeaway do
+
+  TEST_MEAL = 'Mega Breakfast'
```

Comment:
This is fine, but it would be more idiomatic to use

`let(:test_meal) { 'Mega Breakfast }`

same for the other constant